From patchwork Tue Feb  1 20:37:34 2022
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Bryan Brattlof <bb@ti.com>
X-Patchwork-Id: 7528
From: Bryan Brattlof <bb@ti.com>
Subject: [k3conf PATCH 8/8] readme: add j721s2 to the list of supported chips
Date: Tue, 1 Feb 2022 14:37:34 -0600
Message-ID: <20220201203734.26355-9-bb@ti.com>
In-Reply-To: <20220201203734.26355-1-bb@ti.com>
References: <20220201203734.26355-1-bb@ti.com>
Return-Path: nm@ti.com
List-Post: <mailto:linux-patch-review@list.ti.com>
MIME-Version: 1.0
To: TI Linux Patch Review <linux-patch-review@list.ti.com>
Cc: Dave Gerlach <d-gerlach@ti.com>, Nishanth Menon <nm@ti.com>, Bryan
 Brattlof <bb@ti.com>
List-ID: <linux-patch-review@list.ti.com>
List-Post: <mailto:linux-patch-review@list.ti.com>

k3conf now has support for the j721s2. Update the README to reflect this

Signed-off-by: Bryan Brattlof <bb@ti.com>
---
 README.md | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/README.md b/README.md
index 95704d6f94ca0..32896d6bec987 100644
--- a/README.md
+++ b/README.md
@@ -23,8 +23,8 @@ dimension. Use at your own risk. And keep the reset button in reach.
 SUPPORT:
 --------
 
-K3CONF currently supports AM654, J721E, J7200 and AM64x Devices. Legacy OMAP and DRA7
-platforms are not supported.
+K3CONF currently supports AM654, J721E, J7200, J721S2 and AM64x Devices. Legacy
+OMAP and DRA7 platforms are not supported.
 
 
 
